src/main/java/uk/nhs/prm/repo/ehrtransferservice/exceptions/EhrResponseTimedOutException.java

```diff
@@ -0,0 +1,13 @@
+package uk.nhs.prm.repo.ehrtransferservice.exceptions;
+
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class EhrResponseTimedOutException extends Exception {
```

Comment:
Just confirming, do we want this to be a checked exception or unchecked? Right now it's checked, I think it would be better so have this extend `RuntimeException`.